klarna-on-demand/build.gradle

```diff
@@ -29,15 +17,24 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-}
 
-dependencies {
-    androidTestCompile ('org.powermock:powermock-easymock-release-full:1.5.6') {
-        exclude group : 'org.hamcrest'
-        exclude module: 'objenesis'
+    sourceSets {
+        androidTest.setRoot('src/test')
     }
+}
 
-    androidTestCompile 'org.easymock:easymock:3.3'
+apply plugin: 'android-unit-test'
```

Comment:
@tomerlevi444 almost. Be sure to go to the site listed in the deployment guide, look at the recently uploaded package and make sure nothing is missing inside. If the guide isn't clear enough, I'll gladly walk you through it tomorrow.